README.md

```diff
@@ -1,358 +1,302 @@
-
-```typescript
+```ts
 const noneValue = option.none;
-const someValue = option.of("value");
+const someValue = option.of('value');
 
-option.getOrElse(() => "default")(noneValue); // "default"
-option.getOrElse(() => "default")(someValue); // "value"
+// Convert Option<T> to T | undefined
+option.toUndefined(noneValue); // => undefined
+option.toUndefined(someValue); // => 'value'
 
-option.getOrElseW(() => 3)(noneValue); // 3
-```
+// Convert Option<T> to T | null
+option.toNullable(noneValue); // => null
+option.toNullable(someValue); // => 'value'
 
-##### Compute and get the value
+// Convert Option<T> with a default value
+option.getOrElse(() => 'default')(noneValue); // => 'default'
+option.getOrElse(() => 'default')(someValue); // => 'value'
 
-The last way to get your value is `match` and it allows you to compute before returning it.
-it takes a two functions, the first one is executed if your Option is `none`, the second one if your `Option` contains some value.
+// Convert Option<T> to T | U with a default value of type U
+option.getOrElseW(() => 3)(noneValue); // => 3: string | number
 
-```typescript
+// Apply a different function on None/Some(...)
 const noneValue = option.none;
 const someValue = option.of(10);
 
 const doubleOrZero = option.match(
-  () => 0, // this is called when your Option is none
-  (n: number) => 2 * n // called when your Option has some value
+  () => 0, // none case
+  (n: number) => 2 * n // some case
 );
 
-doubleOrZero(noneValue); // 0
-doubleOrZero(someValue); // 20
-```
-
-### <a name="either"></a>Either
-
-An Either represents a computation that can have two results, called branches or tracks (left and right).
-
-Most of the time, we use it to represent a **computation that can fail**.
-So the left branch represents the failure, and the right branch the success. So when we manipulate an Either, if we enter the left branch, we most of the time won't carry out further manipulations, and just return the Error context that is encapsulated in the left branch (it is still accessible tho, through different helpers). If we are in the right branch, we'll keep on manipulating the value and passing it through the right branch.
-
-An Either is typed `Either<E, A>` where `E` is the type of the left track (E for `Error` most of the time) and `A` the right track.
+doubleOrZero(noneValue); // => 0
+doubleOrZero(someValue); // => 20
 
-#### <a name="build-either"></a>Building an Either
-
-##### From a value
-
-The easiest way to build an `Either` is to use the right or left constructor that returns a value encapsulated as a right or left `Either`.
-
-```typescript
-const leftValue = either.left("value"); // -> you are on the left branch
-const rightValue = either.right("value"); // -> you are on the right branch
+// Pro-tip: option.match is short for the following:
```

Comment:
not sure I would keep this in the cheatsheet

Reply:
I agree it has great value. Maybe we could move it to the details page or have some training exercises on this specific topic